ppr-ui/src/components/collateral/EditCollateral.vue

```diff
@@ -147,61 +164,49 @@ export default defineComponent({
   },
   emits: ['addEditVehicle', 'resetEvent'],
   setup (props, context) {
-    console.log(props.activeIndex)
-    const { setAddCollateral } = useActions<any>(['setAddCollateral'])
-    const { getAddCollateral } = useGetters<any>(['getAddCollateral'])
-
-    const localState = reactive({
-      // eslint-disable-line
-      vehicleTypes: VehicleTypes,
-      vehicle: computed(() => {
-        const vehicles: VehicleCollateralIF[] = getAddCollateral.value.vehicleCollateral
-        if (props.activeIndex >= 0) {
-          return vehicles[props.activeIndex]
-        }
-        return { id: -1, type: '', year: 2021, make: '', model: '', serialNumber: '' }
-      })
-    })
-    const getYears = computed(function () {
-      const year = new Date().getFullYear()
-      return Array.from(
-        { length: year - 1900 },
-        (value, index) => 1901 + index
-      ).reverse()
-    })
+    const {
+      // @ts-ignore - returned by toRef
```

Comment:
Do you know if you can return the whole object instead of the toRef, and then dereference it in the return for setup or not? Might be a way to get around the lint unless that breaks the reactivity

Reply:
Yes, I need the reactivity........ trust me I tried it 10 other ways